### ISSUE TITLE
Fieldwork: Bizon super does not stop when full after first unload

### ISSUE THREAD

**Issue by Voidheartd** [Courseplay/courseplay#4356]
Bizon super on fieldwork course. Manually controlled tractor unloading. May only affect the Bizon super as I was not able to reproduce with a CR10.90 I borrowed for a contract.

I've found that after the first time the combine stops due to being full, it will not stop when full again. The header shuts off but the combine keeps driving the course. It doesn't even show the message about needing to be unloaded. I tried activating stop to unload but that doesn't help. It persists even after stopping CP to unload and reposition then resuming.

There seems to be a related issue with the stop to unload function. It worked the first time I unloaded after enabling the function but wouldn't work after that.

Also, a question, if the combine reaches the end of a row and is being unloaded shouldn't it wait to turn until it's empty? The hired worker behaves this way. This means I get very little unloaded if I arrive at the combine when it gets near the end of a row, leading to the combine getting full with the pipe over the crop and requiring the combine to stop and move out from the crop to clear the pipe for unloading. That takes way longer than just stopping at the end of a row and finishing the unload.

**Comment by pvaiko** [comment]
Log with debug channel 14 please.

**Comment by pvaiko** [comment]
(I can't reproduce this)

**Comment by Voidheartd** [comment]
I'll work on that. Any answer to the question about unloading before turning?

**Comment by pvaiko** [comment]
Sorry, yes, it should, I think there's in issue open about it already, if not, I'd prefer having it in a separate issue.

**Comment by Voidheartd** [comment]
Yeah, I just wanted to check if it was intended or not before opening an issue. I had checked but couldn't find an issue for it.

**Comment by Voidheartd** [comment]
I had a time trying to reproduce as after the first time I unloaded the combine it kept doing the space clearing manuever and then saying it was full despite having almost nothing in it. It eventually just refused to move if there was only 1 row between it and the field edge. I had to manually harvest the row so that I could start it on the next row and get it to actually work again.

After dealing with that nonsense I finally was able to reproduce the bug. The combine got full and just kept driving.
[log.txt](https://github.com/Courseplay/courseplay/files/3723331/log.txt)
